doc/EuPathDB_AnnotationHub_Tutorial.Rmd

```diff
@@ -114,7 +114,7 @@ Hmm it appears that something at annotationhub is being updated and all my queri
 returned 403, Forbidden.  I will for the moment stop the following blocks from running.
 
 ```{r 04get_orgdb, eval=FALSE}
-chosen_ah <- res[[2]]
+chosen_ah <- res[[1]]
```

Comment:
Very minor suggestion, but it may be useful to create a "constant" variable describing what the index refers to (rather than just dropping in some number which may not be obvious to the reader), e.g. something along the lines of  `SAMPLE_ID_INDEX <- 1`.

Another thing you could consider doing, either along-side of this, or in-place of it, would be to print some kind of previous of res (or include it in a comment), so it is clear what the "1" refers to.